### PR TITLE
Normalize line endings to lf

### PR DESCRIPTION
**Pull request by Shredder121**
'Backport' of #1711

### CODE REVIEW

File: querydsl-codegen/src/main/java/com/mysema/query/codegen/AnnotationHelper.java [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
-    /**
-     * Get the {@link TypeCategory} according with object Class and Annotation.
-     * @param cl Class of type.
-     * @param annotation Annotation found on element.
-     * @return Custom {@link TypeCategory}.
-     */
-    TypeCategory getTypeByAnnotation(Class<?> cl, Annotation annotation);
-}
+/*
+ * Copyright 2015, The Querydsl Team (http://www.querydsl.com/team)
```
Comment (johnktims) [inline querydsl-codegen/src/main/java/com/mysema/query/codegen/AnnotationHelper.java]: Should this be bumped to 2016?

**Comment by timowest** [comment]
Looks good

File: querydsl-codegen/src/main/java/com/mysema/query/codegen/AnnotationHelper.java (hunk 1 above)
Comment (Shredder121) [inline querydsl-codegen/src/main/java/com/mysema/query/codegen/AnnotationHelper.java]: I don't think that's necessary, nothing has changed.
What does @timowest think?

Comment (timowest) [inline querydsl-codegen/src/main/java/com/mysema/query/codegen/AnnotationHelper.java]: I don't think the bumping is necessary

**Comment by johnktims** [comment]
LGTM

**Comment by Shredder121** [comment]
Thanks!

**Pull request merged**